cmake/FindLpeg.cmake

```
@@ -1,4 +1,13 @@
-find_library2(LPEG_LIBRARY NAMES lpeg_a lpeg liblpeg_a lpeg${CMAKE_SHARED_LIBRARY_SUFFIX} PATH_SUFFIXES lua/5.1)
+find_library2(LPEG_LIBRARY NAMES lpeg_a lpeg liblpeg_a lpeg.so lpeg${CMAKE_SHARED_LIBRARY_SUFFIX} PATH_SUFFIXES lua/5.1)
+if(CMAKE_SYSTEM_NAME STREQUAL "Darwin" AND LPEG_LIBRARY MATCHES ".so$")
+  execute_process(
+    COMMAND otool -hv "${LPEG_LIBRARY}"
+    OUTPUT_VARIABLE LPEG_HEADER
+    )
+  if(LPEG_HEADER MATCHES ".*BUNDLE.*")
```

```suggestion
  if(LPEG_HEADER MATCHES ".* BUNDLE .*")
```
Maybe safer? Doesn't matter hugely though.